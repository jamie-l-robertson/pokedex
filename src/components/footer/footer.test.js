import React from 'react';
import { Footer } from './';

const wrapper = mount(<Footer />);

describe('<Footer />', () => {
  it('Renders', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Has the correct page Landmark', () => {
    expect(wrapper.find('footer')).toHaveLength(1);
  });
});
