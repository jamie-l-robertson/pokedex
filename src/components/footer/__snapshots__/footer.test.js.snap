// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders 1`] = `
<styled.footer>
  <footer
    className="sc-bdVaJa hCQlbO"
  >
    <p>
      Built by
       
      <a
        href="https://www.linkedin.com/in/jamie-robertson-8b75905b/"
        rel="noopener noreferrer"
        target="_BLANK"
      >
        Jamie Robertson
      </a>
       
      with
       
      <a
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_BLANK"
      >
        React
      </a>
       
      and 
      <span>
        â™¥
      </span>
      .
    </p>
    <p>
      Content managed by
       
      <a
        href="https://datocms.com"
        rel="noopener noreferrer"
        target="_BLANK"
      >
        DatoCMS
      </a>
      , images provided by
       
      <a
        href="https://github.com/ZeChrales/PogoAssets"
        rel="noopener noreferrer"
        target="_BLANK"
      >
        PogoAssets
      </a>
    </p>
    <p>
      View project on
       
      <a
        href="https://github.com/ru7hl355/pokedex"
        rel="noopener noreferrer"
        target="_BLANK"
      >
        Github
      </a>
    </p>
  </footer>
</styled.footer>
`;
